src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] ggss) {
+
+        Machine[] machines = {new Truck(), new Excavator(), new Bulldozer()};
```

Comment:
Consider using abstract references instead of specific ones. This will allow you to use polymorphism in your code. For example, instead of `Machine[] machines = {new Truck(), new Excavator(), new Bulldozer()};`, you can use `Machine[] machines = {new Machine(), new Machine(), new Machine()};`.